.krel-bom-config.yaml

```diff
@@ -0,0 +1,28 @@
+---
+namespace: https://k8s.io/release
```

Comment:
Remove these namespaces to let bom generate a unique namespace for each document (this will form the ID of the document and its elements)
```suggestion
```

Reply:
done, thanks PTAL @puerco